src/Shared/Recruit.Shared.Web/ViewModels/ApplicationReview/ApplicationStatusConfirmationViewModel.cs

```diff
@@ -18,6 +18,15 @@ public class ApplicationStatusConfirmationViewModel
 
         public bool ShowStatusSuccessful => Outcome.Value == ApplicationReviewStatus.Successful;
         public bool ShowStatusUnSuccessful => Outcome.Value == ApplicationReviewStatus.Unsuccessful;
+        public string YesMessageText => ShowStatusSuccessful
+            ?
+            "Yes, make this application successful and notify the applicant"
+            :
+            ShowStatusUnSuccessful
+                ? "Yes, notify the applicant"
+                :
+                "Yes";
+        public string NoMessageText => ShowStatusSuccessful ? "No,do not make this application successful" : "No";
```

Comment:
There should be a space after the comma.

Reply:
Done